client/src/components/Applications/AppLayout/AppLayout.jsx

```diff
@@ -51,7 +51,7 @@ const AppsHeader = (categoryDict) => {
 const AppsRoutes = () => {
   const { path } = useRouteMatch();
   const dispatch = useDispatch();
-  const appDict = useSelector((state) => state.apps.appDict, shallowEqual);
+  const htmlDict = useSelector((state) => state.apps.htmlDict, shallowEqual);
```

Comment:
Renamed `appDict` to `htmlDict`, as this object is only used to fast-load HTML type apps